Added benchmark blog post

ecton opened the pull request.

ecton committed:
Added benchmark blog post

daxpedda commented on content/blog/2022-01-18-commerce-benchmark.md:
```diff
@@ -0,0 +1,116 @@
+[![postgresql find product by name](https://khonsulabs-storage.s3.us-west-000.backblazeb2.com/bonsaidb-scaleway-gp1-xs/commerce/large-writeheavy/4/postgresql-FindProduct.png)](https://khonsulabs-storage.s3.us-west-000.backblazeb2.com/bonsaidb-scaleway-gp1-xs/commerce/large-writeheavy/4/index.html#postgresql-FindProduct)
+
+This set of graphs is from the "large, write-heavy, 2 agents per core"
+benchmark, run on a [Scaleway](https://scaleway.com) on a GP1-XS instance
```
```suggestion
benchmark, run on a [Scaleway](https://scaleway.com) GP1-XS instance
```

daxpedda commented:
We should probably not use any `https://github.com/khonsulabs/bonsaidb/tree/main` links because things might change that make the blog post make no sense, just use perma-links instead.

ecton commented:
I double checked -- Zola does have linkchecking built in, and it surprisingly even checks for anchors being present for deep-links into documents! But, I discovered it's only performed when running the "check" command, so I added it to CI. That way any broken links on any of the pages will be reported in CI.

After pondering a bit longer, I decided I agreed with not wanting the hassle of the links breaking on old posts. Technically it'd be a better user experience if we updated the links if they eventually break, but that requires catching the links being broken (only happens manually right now) and that the destination remains valid.

For now, I've just used permalinks as suggested.
